tools/Utils/generate_test_certificates.py

```diff
@@ -289,8 +289,11 @@ def main():
         subprocess.call(["c_rehash", server_dir])
         subprocess.call(["c_rehash", client_dir])
     else :    
-        c_rehash_tools.process_dir(server_dir)
-        c_rehash_tools.process_dir(client_dir)
+        current_working_dir = os.path.dirname(os.path.realpath(__file__))
+        server_dir = os.path.realpath(server_dir)
+        client_dir = os.path.realpath(client_dir)
+        os.system("python " + current_working_dir + "\\c_rehash.py -i " + server_dir)
+        os.system("python " + current_working_dir + "\\c_rehash.py -i " + client_dir)
```

Comment:
@LuxoftAKutsan, it's committed by mistake. Next commit will revert those changes.